WcaOnRails/app/views/delegates_panel/pending_claims_for_subordinate_delegates.html.erb

```diff
@@ -0,0 +1,33 @@
+<% provide(:title, 'Pending claims for subordinate Delegates') %>
+
+<div class="container">
+  <h2>Pending WCA ID claims for Delegates managed by <%= @user.name %></h2>
+
+  <% if @user.subordinate_delegates.empty? %>
+    <div class="well">No subordinate Delegates to display</div>
+  <% else %>
+    <div class="panel-group">
+      <% @user.subordinate_delegates.sort_by { |u| u.users_claiming_wca_id.size }.reverse_each do |delegate| %>
```

Comment:
Do you think we should include pending claims for the senior delegate here as well? Probably not so useful for the senior delegate, but it does seem useful for the Board if they want the full picture of pending WCA ID claims.